sample-app/src/main/java/org/smartregister/pnc/sample/config/PncRegisterQueryProvider.java

```diff
@@ -18,10 +18,10 @@
     @Override
     public String getObjectIdsQuery(@Nullable String filters, @Nullable String mainCondition) {
         if (TextUtils.isEmpty(filters)) {
-            return "SELECT object_id, last_interacted_with FROM ec_client_search WHERE register_type = 'pnc' " +
+            return "SELECT object_id, last_interacted_with FROM ec_client_search " +
                     "ORDER BY last_interacted_with DESC";
         } else {
-            String sql = "SELECT object_id FROM ec_client_search WHERE date_removed IS NULL AND register_type = 'pnc' AND phrase MATCH '%s*'" +
+            String sql = "SELECT object_id FROM ec_client_search WHERE date_removed IS NULL AND phrase MATCH '%s*'" +
                     "ORDER BY last_interacted_with DESC";
```

Comment:
update the table to table specified in `PncMetadata` and for fts table occurrences e.g `ec_client_search`  use `CommonFtsObject.searchTableName(table from Pnc Metadata )`

Reply:
done

Reply:
you can use this `PncUtils.metadata().getTableName()`

Reply:
Done